repose-aggregator/components/filters/compression-filter/src/main/scala/org/openrepose/filters/compression/CompressionFilter.scala

```diff
@@ -91,6 +93,7 @@ class CompressionFilter @Inject()(configurationService: ConfigurationService, co
           logger.error("Error with the CompressingFilter", e)
           httpResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR)
       }
+      httpResponse.commitToResponse()
```

Comment:
The wrapper usage here is unnecessary since in our code we aren't making any changes to the response. By using it you are stating an intention to mutate the response in our code in some way, but you don't, that's handle entirely by the nested filter. All you've really done is introduced a hard break in the stream where one wasn't necessary.